README.md

```diff
@@ -117,6 +117,10 @@ the following actions are performed:
 
 By default, the tracking API writes data into files in a local `./mlruns` directory.
 
+`spacy.MLflowLogger.v1` and below automatically call the default console logger.
+However, starting with `spacy.MLflowLogger.v2`, console logging must be activated
+through the use of the [ChainLogger](#chainlogger).
+
```

Comment:
Maybe a few words on why this is necessary?

Reply:
Done.